docs/tutorials/platform_beginner/5_display_a_list_of_content_items.md

```diff
@@ -6,130 +6,121 @@ In this step you will display a table of all Rides on the front page.
 
 ## Customize the homepage template
 
-In `templates/full/home_page.html.twig` replace the "Hello world" with a call to a controller which will display the list of all existing Rides:
+In `templates/full/home_page.html.twig` replace the "Hello world" with a table that will display the list of all existing Rides:
 
-``` html+twig hl_lines="5"
+``` html+twig hl_lines="5 18"
 {% extends "main_layout.html.twig" %}
 
 {% block content %}
     <div class="col-xs-10 col-xs-offset-1 text-justified">
-        {{ render_esi( controller( 'App\\Controller\\HomepageController::getAllRidesAction' ) ) }}
+        {% for ride in rides.currentPageResults %}
+            {% if loop.first %}
+                <table class="table table-hover">
```

Comment:
Probably what is inside the `loop.first` and `loop.last` ifs doesn't have to be in the for loop.